server/app/views/dev/DebugContent.java

```diff
@@ -0,0 +1,62 @@
+package views.dev;
+
+import static j2html.TagCreator.div;
+import static j2html.TagCreator.p;
+
+import auth.FakeAdminClient;
+import j2html.tags.specialized.DivTag;
+import views.BaseHtmlView;
+
+public class DebugContent extends BaseHtmlView {
```

Comment:
**suggestion (blocking):** add javadoc for this class

Reply:
Done.